### ISSUE TITLE
Quest: Pamela's Doll, npc "Ghost of the Past"

### ISSUE THREAD

**Issue by Cauton** [LightsHope/server#858]
Upon nearing one of the three pieces of Pamela's Doll a "Ghost of the Past" is supposed to spawn. It does, in multitudes. One building contains 2 pieces of Pamela's Doll, spawning 2 ghosts. These ghosts spawn once more in the time a regular level-appropriate player can kill 1. Resulting in infinite spawns.

**Comment by Cabro** [comment]
Yes, confirm, this quest is seriously overtuned. Seriously seriously overtuned. You need 3 players to do this quest, and even then you are lucky if nobody dies.
